plugins/crypto/openssl/ua_openssl_create_certificate.c

```diff
@@ -0,0 +1,365 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ *
+ *    Copyright 2021 (c) Christian von Arnim, ISW University of Stuttgart (for VDW and umati)
+ *
+ */
+
+#include <open62541/plugin/create_certificate.h>
+
+#include "securitypolicy_openssl_common.h"
+
+#if defined(UA_ENABLE_ENCRYPTION_OPENSSL) || defined(UA_ENABLE_ENCRYPTION_LIBRESSL)
+
+
+#define RSA_KEY_SIZE 4096
+#include <openssl/pem.h>
+#include <openssl/x509v3.h>
+
+/**
+ * Join an array of UA_String to a single NULL-Terminated UA_String
+ * separated by character sep
+ */
+static UA_StatusCode join_string_with_sep(
```

Comment:
Fix the alignment.
Use UA_String *strings instead of UA_String strings[].
Do this consistent everywhere.